CONTRIBUTING.md

```diff
@@ -126,6 +126,10 @@ Examples:
   - For a new translation of an existing page: `cp: add Tamil translation`
   - For related changes to several pages: `grep, find, locate: synchronize format of wildcards`
 
+## Inclusive language
+
+Where possible, use inclusive language. We know that this may not always be possible when documenting projects or technologies that keep attributing special meanings to terminology that may be perceived as non-inclusive. Nonetheless, in an effort to be clear and consistent, always document objectively and always assume good faith.
```

Comment:
I think this paragraph, as it's currently written, is too vague to be useful. We would need a couple examples to make it clear what we mean. And I'd suggest dropping the last sentence, which is of course laudable in spirit, but not specifically related to the topic of this section.

Here's an attempt at materializing what I mean. It's just an example — I am open to different wording and phrasing.

> Where possible, use inclusive language in the content of pages. For example, prefer terms like "denylist"/"allowlist" or "blocklist"/"passlist" instead of "blacklist"/"whitelist"; "primary"/"secondary" instead of "master"/"slave"; "them" instead of "him"/"her", and so on.
>
> Of course, this shouldn't sacrifice clarity of the content, such as when documenting tools where such terms have specific technical meanings, and their usage is central to explaining the involved concepts.